config/initializers/dhis2.rb

```diff
@@ -0,0 +1,6 @@
+Dhis2.configure do |config|
```

Comment:
You could move this configuration into the task itself I think, as we don't need DHIS integration anywhere else.  Its pretty isolated.